PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/webapp/DeploymentSlotPropertyView.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.intellij.helpers.webapp;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import com.intellij.openapi.project.Project;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.webapp.deploymentslot.DeploymentSlotPropertyViewPresenter;
+
+public class DeploymentSlotPropertyView extends WebAppBasePropertyView {
+
+    /**
+     * Initialize the Web App Property View and return it.
+     */
+    public static WebAppBasePropertyView create(@NotNull Project project, @NotNull String sid,
+                                                @NotNull String resId, @NotNull String name) {
+        DeploymentSlotPropertyView view = new DeploymentSlotPropertyView(project, sid, resId, name);
+        view.onLoadWebAppProperty(sid, resId, name);
+        return view;
+    }
+
+    private DeploymentSlotPropertyView(@NotNull final Project project, @NotNull final String sid,
+                                       @NotNull final String webAppId, @NotNull final String name) {
+        super(project, sid, webAppId);
+        presenter = new DeploymentSlotPropertyViewPresenter();
```

Comment:
The presenter should be initialized in base class, use an interface like "createPresenter" to do so. Also attach view in super constructor, because related functionalities are defined there.  Commonly, the derived classes only need to implement the interfaces.